curvefs/src/client/fuse_client.h

```diff
@@ -217,6 +226,50 @@ class FuseClient {
     SpaceBaseClient *spaceBase_;
 };
 
+class RenameOperator {
+ public:
+    RenameOperator(uint32_t fsId,
+                   uint64_t parentId,
+                   std::string name,
+                   uint64_t newParentId,
+                   std::string newname,
+                   std::shared_ptr<DentryCacheManager> dentryManager,
+                   std::shared_ptr<InodeCacheManager> inodeManager,
+                   std::shared_ptr<MdsClient> mdsClient);
+
+    CURVEFS_ERROR GetTxId();
```

Comment:
Add a function to print detail RenameOperator.
std::string ToString() {
}

Reply:
> Add a function to print detail RenameOperator.
> std::string ToString() {
> }

fixed.